src/org/jitsi/impl/neomedia/transform/fec/FECTransformEngine.java

```diff
@@ -144,25 +144,32 @@ public FECTransformEngine()
 
         // Assumption: all packets in pkts have the same SSRC
         Long ssrc = findSSRC(pkts);
-        if (ssrc == null)
+        long primarySsrc = getPrimarySsrc(ssrc);
```

Comment:
If ssrc is null, wouldn't we hit an NPE trying to unbox it?

Reply:
good catch, fixed